recipes/jupyterlab-topbar-text/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "jupyterlab-topbar-text" %}
+{% set version = "0.6.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fdcbe4522bf6f79bb66b33746eedab16f6d371b2c4fdc95417b8d0edc7d2e8fc
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - jupyter-packaging >=0.10,<1
+  run:
+    - jupyterlab ~=3.0
+    - jupyterlab-topbar >=0.6,<0.7
+    - python >=3.6
+
+test:
+  commands:
+    - pip check
+    - test -f "${PREFIX}/share/jupyter/labextensions/jupyterlab-topbar-text/package.json"                                    # [unix]
+    - if exist %PREFIX%\\share\\jupyter\\labextensions\\jupyterlab-topbar-text\\package.json (exit 0) else (exit 1)          # [win]
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jupyterlab-contrib/jupyterlab-topbar-text
+  summary: JupyterLab extension to display arbitrary text in the top bar
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jtpio
+    - fcollonval
```

Comment:
cc @fcollonval adding you here since the repo is part of the `jupyterlab-contrib` org.

Reply:
Thanks @jtpio
i agree to be maintainer.